lib/frontend/post-github-configuration.js

```diff
@@ -33,21 +49,28 @@ module.exports = async (req, res) => {
 
       if (role !== 'admin') {
         res.status(401);
-        return res.json({
+        res.json({
           err: `Failed to add subscription to ${req.body.installationId}. User is not an admin of that installation`,
         });
+        return;
       }
     }
 
-    await Subscription.install({
+    const subscription = await Subscription.install({
```

Comment:
we also deleted install?

Reply:
oh dang I missed the `.destroy()` call here, good catch!

Reply:
I didn't delete it, I just capture the output and use it to create an event!

Reply:
Misread!